### DISCUSSION TITLE
"Set IO Pin Constraint" : How to specify exact location of pins groups

### DISCUSSION THREAD

**Discussion by nerriav** [The-OpenROAD-Project/OpenROAD-flow-scripts#1052] [General] [answered]
Tried forcing the location of a specific clock pin by using the following command
`-region up:{llx lly urx ury} `
(as appears in the openroad manual under 'Pin Constraints' section) 

Pull request reference : #1051 

when I run floorplan stage, I get the following :
``` [ERROR STA-0406] set_io_pin_constraint positional arguments not supported. Error: io.tcl, 54 STA-0406 ```

**Comment by vijayank88** [comment, marked as answer]
Check this [comment](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/1051#discussion_r1185963532)

**Comment by oharboe** [comment]
The documentation has been updated to be more clear.
